extensions/wikia/Email/Controller/CategoryAddController.class.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace Email\Controller;
+
+use \Email\EmailController;
+use Email\Check;
+
+class CategoryAddController extends EmailController {
+
+	/** @var \Title */
+	private $categoryPage;
+
+	/** @var \Title */
+	private $pageAddedToCategory;
+
+	public function initEmail() {
+		$pageTitle = $this->request->getVal( 'pageTitle' );
+		$titleNamespace = $this->request->getVal( 'namespace', NS_MAIN );
+		$categoryPageId = $this->getVal( 'childArticleID' );
+
+		$this->pageAddedToCategory = \Title::newFromText( $pageTitle, $titleNamespace );
```

Comment:
I don't think this should be too much of a problem since I assume the majority of categories will be added after a page has already been created, but I can add a check to the master DB just in case since we have run into that problem with previous emails.